content/en/real_user_monitoring/browser/monitoring_page_performance.md

```diff
@@ -147,6 +147,49 @@ The RUM SDK automatically monitors frameworks that rely on hash (`#`) navigation
 
 ## Add your own performance timing
 
+### Custom Vitals
+
+The `customVital` API allows you to measure the performance of your application at the component or the user flow level. 
```

Comment:
... "at the component level. You can use custom vitals to measure how quickly part of your page renders, or how long it takes for a component to respond to a user interaction."